olive/passes/onnx/moe_experts_distributor.py

```diff
@@ -43,6 +43,7 @@
 
 
 def _dump_graph(node: Message, filepath: str):
+    Path(filepath).parent.mkdir(parents=True, exist_ok=True)
```

Comment:
Double check: do you expect to create folder in **parent folder** of **parent folder** of `filepath`? `cd ../, mkdir -p xxx`?

Reply:
it seems the parent folder is already created before calling the internal function?

Reply:
Yes, not sure it is by design or something else. Please help double confirm.

Reply:
_filepath_ is a path to a file not a directory. So _Path(filepath).parent_ would be a directory. _mkdir_ on that seems correct.

Create the directory in which to create the file. I don't see an issue here.

Reply:
The line seems needless by considering the parent folder is already created when the function is called. What's the scenario that need this line?

Reply:
The function is called from _run_ function and I don't see any calls to _mkdir_. `_dump_graph` is a debug utility function and I use it as such, for debugging. Having the call in one place to create the parent folders before writing out the file doesn't seem wrong.